CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# How to contribute
+
+I'm really glad you're reading this, because if you are then you have shown an interest in helping make LinuxGSM great.
+
+If you haven't already, come find us on [Discord](https://gameservermanagers.com/discord). From there you will have contact with other contributers of the project. We want you working on things you're excited about.
+
+Here are some important resources:
+
+  * [Issues Page](https://github.com/GameServerManagers/LinuxGSM/issues) provides a list of areas that could use some work,
+  * [Developing LGSM](https://github.com/GameServerManagers/LinuxGSM/wiki/Developing-LGSM) gives a detailed guide on developing LGSM,
+  * [LGSM Exit Codes](https://github.com/GameServerManagers/LinuxGSM/wiki/LGSM-Exit-Codes) describes and gives an explanation for exit codes,
+  * [gsquery](https://github.com/GameServerManagers/LinuxGSM/wiki/gsquery.py) describes the uses of the gsquery.py file, and
+  * [Branching](https://github.com/GameServerManagers/LinuxGSM/wiki/Branching) is our final guide to submitting changes.
+  
+## Testing
+
+Please make sure all the code you write is working properly **before** you create a pull request. Information on debugging can be found in the following document:
+[Debug Command](https://github.com/GameServerManagers/LinuxGSM/wiki/debug)
```

Comment:
Are you suggesting removing line 18 then? There is no wiki page on dev-debug except for the page I list directly below.